packages/os/healthdog.timer

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Scheduled Healthdog Pings
+
+[Timer]
+# Don't run missed executions
+Persistent=false
+# Run 120 seconds after boot for the first time
+OnBootSec=120
```

Comment:
```suggestion
OnStartupSec=120
```
This is relative to the init system rather than the hardware boot, which seems like a more logical thing to think about for me - if an initrd or kernel startup takes longer, it doesn't mean we want to give less time for services to start before checking them.

Reply:
Addressed in [f18d841](https://github.com/bottlerocket-os/bottlerocket/compare/ec0abbf943fe3806294f53fd4e118be3d1f529fc..f18d841ee1c2b35d5f09136ea3f7136e2a4d65ae)